test/withStyles_test.jsx

```diff
@@ -55,8 +60,10 @@ describe('withStyles()', () => {
 
   describe('HOC', () => {
     it('creates the styles', () => {
-      function MyComponent() {
-        return null;
+      class MyComponent extends React.Component {
+        render() {
```

Comment:
Why convert to class?

Reply:
SFCs don't work in React 0.13, and I was hoping to make the tests pass on it (only one fails rn)

Reply:
Although actually I think I have a cleaner solution that will let us use SFCs inline, and reduce these diffs; i'll get that up in a few days.

Reply:
Posted https://github.com/airbnb/enzyme/pull/1383; once merged and released, I'll use it here. The diff is much much smaller.